specs/enhance_gluster_cluster_sync_logic.adoc

```diff
@@ -0,0 +1,307 @@
+= Enhance gluster cluster synchronization logic as per recommendations
+
+The intent of this change is to accommodate additional details available as part
+of `get-state` output while syncing the gluster cluster details and store them
+in central store.
+
+Also instead of using `gstatus` for cluster health and volume utilization
```

Comment:
@nthomas-redhat Ack.. This has been done on existing plugins as in https://github.com/Tendrl/node-monitoring/pull/79 but yes that PR will eventually have to land on node-agent rather than node-monitoring(that its currently on)